lib/src/op_store.rs

```diff
@@ -369,6 +369,20 @@ pub struct Operation {
     pub view_id: ViewId,
     pub parents: Vec<OperationId>,
     pub metadata: OperationMetadata,
+    /// Mapping from new commit to its predecessors, or `None` if predecessors
+    /// weren't recorded when the operation was written.
+    ///
+    /// * `commit_id: []` if the commit was newly created.
+    /// * `commit_id: [predecessor_id, ..]` if the commit was rewritten.
+    ///
+    /// This mapping may contain transitive predecessors if a commit was
+    /// rewritten multiple times within the same transaction.
```

Comment:
Could you clarify here *how* it contains such transitive predecessors? It's unclear to me if this is saying that if X was rewritten Y and then to Z, this mapping will contain X->Y and Y->Z, or if it will contain only X->Z, or maybe all of them.

Reply:
The former. Does this looks good?
```
    /// This mapping preserves all transitive predecessors if a commit was
    /// rewritten multiple times within the same transaction. For example, if
    /// `X` was rewritten as `Y`, then rebased as `Z`, these modifications are
    /// recorded as `{Y: [X], Z: [Y]}`.
```

Reply:
Yes, sounds good. Thanks!